artifacts/networkneighborses/01-example.yaml

```
@@ -0,0 +1,38 @@
+apiVersion: spdx.softwarecomposition.kubescape.io/v1beta1
+kind: NetworkNeighbors
+metadata:
+  name: deployment-nginx
+  labels:
+      "kubescape.io/workload-api-group": "spdx.softwarecomposition.kubescape.io"
+      "kubescape.io/workload-api-version" : "v1beta1"
```

chore: fix extra space

```suggestion
      "kubescape.io/workload-api-version": "v1beta1"
```